tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -74,6 +75,12 @@ protected function setUp() {
       'id' => 'civicrm_webform_test',
       'title' => 'CiviCRM Webform Test',
     ]);
+    //Create civi contact for rootUser.
+    $this->utils->wf_civicrm_api('UFMatch', 'create', [
+      'uf_id' => $this->rootUser->id(),
+      'uf_name' => $this->rootUser->getAccountName(),
+      'contact_id' => $this->createIndividual()['id'],
+    ]);
   }
```

Comment:
Awesome - somehow on my local testing and on github actions rootUser had no UF Match - totally ok to ensure this manually 👍

Reply:
That previous commit solves it - awesome